tests/echo_service_tests.rs

```diff
@@ -0,0 +1,40 @@
+use rusk::services::echoer::basic_proto::echoer_client::EchoerClient;
+use rusk::services::echoer::basic_proto::echoer_server::EchoerServer;
+use rusk::services::echoer::basic_proto::EchoRequest;
+use rusk::services::echoer::Rusk;
+use tonic::transport::Server;
+
+pub mod basic_proto {
+    tonic::include_proto!("basic_proto");
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[tokio::test(threaded_scheduler)]
+    async fn echo_works() -> Result<(), Box<dyn std::error::Error>> {
+        let addr = "[::1]:50051".parse()?;
+        let rusk = Rusk::default();
+
+        tokio::spawn(async move {
+            Server::builder()
+                .add_service(EchoerServer::new(rusk))
+                .serve(addr)
+                .await
+                .unwrap()
```

Comment:
I think you should be able to replace this `unwrap()` with `?`.

Reply:
This is created inside a tokio::spawn call. I don't quite see how to declare a closure or something similar there so I can express that it returns a result. 
Let me see if I can do something there since the `?` does not work because the thread declaration doesn't state any `Result` as return type.